kncaputo/urban-stargazer
# Clean up

## What's this PR do?
- Adds gifs to assets
- Cleans up code/ linting/ removes unused vars
- Adds media queries for responsiveness
- Exports modal styles
- Styles modal, including for devices - closes #17 
- Binds modal to app for screen reader accessibility - closes #16 
- Adds styles for router active link

## Where should the reviewer start?
- Main thing to review is media queries in style files

## How should this be manually tested?
- Test in browser by checking out responsiveness across devices
- Click nav links and notice styling
